### PR TITLE
Fix version in required projects. Update recordings

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.csproj [hunk 1]
```diff
@@ -67,7 +67,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (chandrasekarsrinivasan) [inline src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.csproj]: Version has to be changed here

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.cs [hunk 2]
```diff
@@ -32,39 +32,39 @@ public VirtualNetworkTests(Xunit.Abstractions.ITestOutputHelper output)
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
-        [Trait(Category.Owner, Category.brooklynft)]
+        [Trait(Category.Owner, Category.sdnnrp)]
```
Comment (chandrasekarsrinivasan) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.cs]: Can you pull the latest? this is already updated.

File: src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.Netcore.csproj [hunk 3]
```diff
@@ -27,7 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Dns" Version="2.2.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.1.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.1.0-preview" />
```
Comment (chandrasekarsrinivasan) [inline src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.Netcore.csproj]: Can you delete local feed packages for this? Also, I am assuming you are generating this from the latest swagger branch?

Comment (EvgenyAgafonchikov) [reply src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.Netcore.csproj]: Rebased on the target branch.
No, this PR just keeps the version that is already in LocalFeed

**Comment by sphibbs** [comment]
merging to unblock nrp

**Pull request merged**